renderer/components/Wallet/WalletButtons.js

```diff
@@ -1,23 +1,32 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { Box } from 'rebass/styled-components'
-import { FormattedMessage } from 'react-intl'
+import { FormattedMessage, injectIntl } from 'react-intl'
 import { Button } from 'components/UI'
 import messages from './messages'
+import { intlShape } from '@zap/i18n'
 
-const WalletButtons = ({ openModal }) => (
+const WalletButtons = ({ intl, openModal, totalBalance }) => (
```

Comment:
Since this is a functional component i'd advice to use `useIntl` hook instead of `inject`

Reply:
Done.